migrations/app/20170619113855_ldu_capacity_view.js

```diff
@@ -3,9 +3,6 @@ exports.up = function (knex, Promise) {
       WITH SCHEMABINDING 
       AS 
       SELECT SUM(total_points) AS total_points
-        , SUM(sdr_points) AS sdr_points
-        , SUM(sdr_conversion_points) AS sdr_conversion_points
-        , SUM(paroms_points) AS paroms_points
```

Comment:
After chatting to Daryl I think another migration file would be the best scenario.